modules/search/src/service.rs

```diff
@@ -84,4 +84,42 @@ impl SearchService {
                 .collect(),
         })
     }
+
+    // `filters` should be of the form, "full text search({field}{op}{value})*", e.g.
+    // "some text&published>=2020/11/11&location=localhost&severity=low|high&modified=true"
+    pub async fn search_sboms<'a>(
```

Comment:
I'd hoped this would just be `graph.sboms(...same params...)` which could return the entire collection or a filtered subset.

Reply:
Not necessarily `locate_sbom(...)` but perhaps a new `.sboms(...)` method to do the fetch/filter.

Reply:
Hm, I can't seem to find a method with that name.

Reply:
Yeah, we probably need to write it.